src/libs/services/NetService.js

```diff
@@ -46,9 +47,11 @@ const handleOffline = () =>
 
 const toggleNetWatcher = makeNetWatcher()
 
-export const NetService = {
+const NetService = {
   handleOffline,
-  isConnected,
-  isOffline,
-  toggleNetWatcher
+  isConnected: devConfig.isOffline ? isOffline : isConnected,
+  isOffline: devConfig.isOffline ? isConnected : isOffline,
```

Comment:
My understanding is that `devConfig.isOffline` is just inverting the result here. So the app would consider you to be online if you temporarily lose the connexion (although highly improbable). Is that intented?